versions.js

```diff
@@ -17,6 +17,7 @@
 const versionsPerRelease = new Map([
   // This is a mapping from Chromium version => Puppeteer version.
   // In Chromium roll patches, use 'NEXT' for the Puppeteer version.
+  ['92.0.4512.0', 'v9.1.1'],
```

Comment:
This should have said `'NEXT'` per the comment above (sorry for missing this). Could you please patch this before cutting the release?

Reply:
As a separate cl or in this one?

Reply:
Ah, sorry. Got confused by the patches